testapps/testapp/src/main/java/com/microsoft/identity/client/testapp/MsalWrapper.java

```diff
@@ -166,9 +85,13 @@ public void acquireToken(final Activity activity,
                                         .split(" ")
                         )
                 )
-                .withCallback(getAuthenticationCallback(notifyCallback));
+                .withCallback(getAuthenticationCallback(callback));
 
-        if (USE_POP) {
+        if (requestOptions.getAuthority() != null && !requestOptions.getAuthority().isEmpty()) {
+            builder.fromAuthority(requestOptions.getAuthority());
+        }
+
+        if (requestOptions.getAuthScheme() == Constants.AuthScheme.POP) {
             try {
                 builder.withAuthenticationScheme(
                         PoPAuthenticationScheme.builder()
```

Comment:
Can we add another `Spinner` and `EditText` to accept the `HttpMethod` and `Resource URL`? These could _optionally_ be shown only when the Auth Scheme `Spinner` indicates `PoP`

Reply:
sure np